nurseconnect/forms.py

```diff
@@ -0,0 +1,144 @@
+from django import forms
+from django.contrib.auth.models import User
+from django.core.validators import RegexValidator
+from django.utils.translation import ugettext_lazy as _
+
+ZATEL_REG = r"^((?:\+27|27)|0)[\s-]?(\d{2})[\s-]?(\d{3})[\s-]?(\d{4})[\s]*$"
+INT_PREFIX = "+27"
+
+
+class RegistrationForm(forms.Form):
+    username = forms.CharField(
+        required=False,
+        validators=[
+            RegexValidator(
+                ZATEL_REG,
+                "Please enter a valid South African telephone number"
+            )
+        ],
+        widget=forms.TextInput(
+            attrs={
+                "placeholder": "eg. 0821234567",
+                "type": "tel",
+                "zaTel": "true",
+                "required": "required"
+            }
+        ),
+        label=_("Mobile Number"),
+    )
```

Comment:
The nesting there is slightly too deep. How about separating the fields with open lines for readability?